src/components/index/GettingStarted.js

```diff
@@ -0,0 +1,87 @@
+/********************************************************************************
+ * Copyright (C) 2020 TypeFox and others.
```

Comment:
minor: unless the file is completely copied I would use a new header:

```suggestion
 * Copyright (C) 2022 EclipseSource and others.
```